src/core_plugins/kibana/public/discover/_discover.scss

```diff
@@ -0,0 +1,306 @@
+
+  // SASSTODO: add a monospace modifier to the doc-table component
+  tbody {
+    font-family: 'Lucida Console', Monaco, monospace;
+  }
+}
+
+// SASSTODO: replace the padding value with a variable
+.dscTable__footer {
+  background-color: $euiColorLightShade;
+  padding: 5px 10px;
+  text-align: center;
+}
+
+// SASSTODO: seems like this class is not used anywhere
+.discover-table-details-toggle {
+  margin-bottom: 3px;
+}
+
+.dscTable__timefield {
+  white-space: nowrap;
+}
+
+/**
+ * 1. When switching between open and closed, the toggle changes size
+ *    slightly which is a problem because it forces the entire table to
+ *    re-render which is SLOW.
+ */
+.dscTable__openButton {
+  appearance: none;
+  background-color: transparent;
+  padding: 0;
+  border: none;
+  width: 14px; /* 1 */
+  height: 14px;
+}
+
+// SASSTODO: seems like this class is not used anywhere
+.discover-field-filter {
+  background-color: $euiColorLightShade;
+  margin-right: 10px;
+
+  .form-group {
+    margin-bottom: 0;
+  }
+}
+
+/**
+ * 1. Override sidebar-item-title styles.
+ */
+.dscSidebarItem {
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-top: 0 !important; /* 1 */
+  padding-bottom: 0 !important; /* 1 */
+  height: $euiSizeXL;
+
+  &--active {
+    background: shade($euiColorLightestShade, 5%);
+    color: $euiColorFullShade;
+    font-weight: bold;
+  }
+
+  &:hover,
+  &:focus {
+    .dscSidebarItem__action {
+      opacity: 1;
+    }
+  }
+}
+
+/**
+ * 1. Truncate long text so it doesn't push the actions outside of the container.
+ */
+.dscSidebarItem__label {
+  overflow: hidden; /* 1 */
+  text-overflow: ellipsis; /* 1 */
+}
+
+/**
+ * 1. Only visually hide the action, so that it's still accessible to screen readers.
+ * 2. When tabbed to, this element needs to be visible for keyboard accessibility.
+ */
+.dscSidebarItem__action {
+  opacity: 0; /* 1 */
+
+  &:focus {
+    opacity: 1; /* 2 */
+  }
+}
+
+// SASSTODO: replace the padding value with a variable
+.dscFieldDetails {
+  padding: 10px;
+  background-color: shade($euiColorLightestShade, 5%);
+  color: shade($euiColorEmptyShade, 65%);
+
+  // SASSTODO: add an option or a modifier to the progressbar component instead
+  .progress {
+    background-color: shade($euiColorLightestShade, 5%);
+    margin-bottom: 0;
+    border-radius: 0;
+
+    // SASSTODO: replace the padding and line-height values with variables
+    .progress-bar {
+      padding-left: 10px;
+      text-align: right;
+      line-height: 20px;
+      max-width: 100%;
+
+      // SASSTODO: replace the padding and margin values with variables
+      span {
+        background-color: rgba(0, 0, 0, 0.8);
+        padding: 3px;
+        border-radius: $euiBorderRadius;
+        margin-left: 3px;
+        margin-right: 3px;
+      }
+    }
+  }
+}
+
+// SASSTODO: seems like this class is not being used anywhere
+.discover-field-details-count {
+  white-space: nowrap;
```

Comment:
I'd remove this and replace it with .eui-textNoWrap in the template.

https://elastic.github.io/eui/#/utilities/css-utility-classes